sources/packages/api/src/route-controllers/designation-agreement/designation-agreement.controller.ts

```diff
@@ -16,9 +17,10 @@ import {
   UserToken,
 } from "../../auth/decorators";
 import { DesignationAgreementService, FormService } from "../../services";
-import { getUTCNow } from "../../utilities";
+import { getDateOnlyString, getUTCNow } from "../../utilities";
 import {
   GetDesignationAgreementDto,
+  GetDesignationAgreementsDto,
```

Comment:
IMO, while naming the DTOs we generally avoid Get and use them commonly for method names.

Reply:
Actually the original idea was express in the DTOs the purpose input/output. Usually the input ones requires some class validators and will require to be a class instead of an interface, hence having the same name will be tricky.
We have quite a few starting with Get. Thoughts?